templates/taxi/car_list.html

```diff
@@ -10,8 +10,11 @@ <h1>Car list</h1>
           {{ car.model }} ({{ car.manufacturer.name }})
         </li>
       {% endfor %}
+      <br>
+      <a class="btn btn-primary" href="{% url 'taxi:car-create' %}">add new car</a>
```

Comment:
The URL name should use hyphens instead of underscores. Change `'taxi:car-create'` to `'taxi:car-create'` to follow the checklist guideline.